src/elements/content-sidebar/SidebarNav.js

```diff
@@ -66,8 +68,9 @@ const SidebarNav = ({
     isOpen,
     onNavigate,
     onPanelChange = noop,
+    signSideBarProps,
 }: Props) => {
-    const { enabled: hasBoxSign } = useFeatureConfig('boxSign');
+    const { enabled: hasBoxSign } = signSideBarProps || {};
```

Comment:
is there a permission via the user that enables this feature too ?

Reply:
yes, there is a user permission that enables/disables sign controlled by the admin